_schemaV2.graphql

```diff
@@ -9183,6 +9183,36 @@ type CreatePartnerContactSuccess {
   partnerContact: Contact
 }
 
+type CreatePartnerLocationDaySchedulesFailure {
```

Comment:
An example of how the UI could look, seems nice to create the full schedule in one call 🦑 


https://github.com/user-attachments/assets/c7175483-2699-4db3-b156-0ade4ae9d0c5

Reply:
Have to do a bit of faffing on both sides as the API expecting time as integer (seconds from 12am)